packages/frontend2/src/app/(legacy)/faq/get-faq-items.tsx

```diff
@@ -0,0 +1,433 @@
+import React, { type ReactNode } from 'react'
+import { OutLink } from '~/app/_components/out-link'
+
+export interface FaqItem {
+  question: string
+  answer: React.ReactNode
+}
+
+export function getFaqItems(): FaqItem[] {
```

Comment:
Let's make that a const with markdown answers, so:
- glossary links can pick it up
- we don't have hardcoded ReactNodes